rules/private/phases.bzl

```diff
@@ -30,24 +35,22 @@ phase_coverage_jacoco = _phase_coverage_jacoco
 
 phase_ijinfo = _phase_ijinfo
 
-phase_noop = _phase_noop
+phase_javainfo = _phase_javainfo
```

Comment:
Looks like these were sorted alphabetically. Is that what is going on here?

Reply:
That's right.